cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/utils/Properties.java

```diff
@@ -28,22 +28,40 @@
 
 import org.apache.cassandra.spark.data.FileType;
 
-public class Properties
+public final class Properties
 {
-    public static final int ONE_MEBIBYTE = 1024 * 1024;
+    private static final int SEC = 1000;
+    private static final int MIN = 60;
+    private static final long KB = 1024;
```

Comment:
this is actually kibibyte
```suggestion
    private static final long KiB = 1024;
```

Reply:
I really don't like what SI did with binary units, to be honest. Besides that, `KiB` would trigger CheckStyle, and would have to be changed to either `KIB` or even `KI_B`, which looks confusing, in my opinion. Can we please keep these as they are?

Reply:
Cassandra codebase is very explicit about these units, so I think we should be consistent with that.

Reply:
Renamed.